Makefile

```diff
@@ -157,7 +157,7 @@ gen-k8s-check: $(apis_sources)
 	touch $@
 
 components:
-	./hack/components.sh
+	./hack/components/components.sh
```

Comment:
You need to change this and the location of the script in the same commit.

Reply:
not sure what you mean. I did change the this and the location in the same commit. can you elaborate?

Reply:
Now it looks fine, either github tricked me or I missed it.

Reply:
ack, changing to resolved